samples/TEE_aware/dotnet/README.md

```diff
@@ -81,8 +81,7 @@ To learn more about writing TEE aware applications, please read the [TEE aware a
 
 ## Running the sample
 
-To run the sample in package mode, use `make run` to launch both the server and the local client that will
-send images located inside `test_samples` for inferencing.
```

Comment:
This line is correct. It says that the images inside the `test_samples` directory will be sent for inferencing.

Reply:
I think this dotnet sample doesn't have this functionality?